css-typed-om/Overview.bs

```diff
@@ -1654,29 +1653,6 @@ of all the "math" operations.
     it returns a new {{CSSMathInvert}} object.
 </div>
 
-<div algorithm>
-    The <dfn>[=type=] of a CSSMathValue</dfn>
-    depends on its class:
-
-    <dl class=switch>
-        : {{CSSMathSum}}
-        : {{CSSMathMin}}
-        : {{CSSMathMax}}
-        :: The [=type=] is the result of [=adding=] the [=types=]
-            of each of the [=list/items=] in its {{CSSMathSum/values}} internal slot.
-
-        : {{CSSMathProduct}}
-        :: The [=type=] is the result of [=multiplying=] the [=types=]
-            of each of the [=list/items=] in its {{CSSMathProduct/values}} internal slot.
-
-        : {{CSSMathNegate}}
-        : {{CSSMathInvert}}
-        :: The [=type=] is the same as the [=type=] of its {{CSSMathNegate/value}} internal slot,
-            but with all [=map/values=] negated.
-    </dl>
-</div>
```

Comment:
No, all of this needs to stay - it defines the type of a CSSMathValue. We just need to remove the `.type` member in the IDL, since that's carried by the superclass.  The superclass's definition of `.type` will then figure out how to translate the type into some JS object.

Reply:
I think there's actually two we need to remove: CSSMathValue.type and CSSUnitValue.type?

Reply:
Yeah.

Reply:
Ahhh I for some reason thought this had been copied twice. :/ Gotcha. Making the change. g